std/assembly/array.ts

```diff
@@ -27,11 +27,20 @@ import {
   MAX_DOUBLE_LENGTH
 } from "./internal/number";
 
+@inline
+function mix<T extends number>(a: T, b: T): T {
```

Comment:
To me, it's not quite obvious what this is doing just from its name. Would favor to not use a helper here.

Reply:
Done!